### ISSUE TITLE
Galleries disappeared

### ISSUE THREAD

**Issue by bgfrancey** [miromannino/Justified-Gallery#107]
Hi there -
I have been using Justified Gallery for a while with no issue on a clients site. I made a small adjustment to the settings today - checking off the right-click option so photos couldn't be saved and suddenly all the galleries except one disappeared. I'm not sure if this is a bug but wanted to let you know.

**Comment by miromannino** [comment]
Can you give me the website to inspect how you call it and if there are errors on the console?
Thanks

**Comment by bgfrancey** [comment]
Sure - it's http://www.sharronmatthews.com/gallery/. There were no errors on the console when I changed the setting. They just disappeared.

**Comment by miromannino** [comment]
Yes, I've got the error, but seems about the server. For the URL:

http://www.sharronmatthews.com/wp-content/plugins/flickr-justified-gallery/js/flickrJustifiedGalleryWPPlugin.js?ver=3.4.0

It returned a 503 (service unavailable). Maybe you've found some server problems for that?

Thank you

**Comment by bgfrancey** [comment]
We had a server issue recently but I’m pretty sure it’s resolved now.
